docs/snippets/vue/list-story-unchecked.vue3.js.mdx

```diff
@@ -0,0 +1,21 @@
+```js
+// List.stories.js
+
+import List from './List.vue';
+import ListItem from './ListItem.vue';
+
+//👇 Imports a specific story from ListItem stories
+import { Unchecked } from './ListItem.stories';
+
+export const OneItem = (args) => ({
+  components: { List, ListItem },
+  setup() {
+    //👇 The args will now be passed down to the component
```

Comment:
We should probably say `template` here instead of component (and in each of the vue3 stories)
```suggestion
    //👇 The args will now be passed down to the template
```

Reply:
@phated fixed with the latest commit. Thanks for the clarification.